blazor/predefined-dialogs/getting-started.md

```diff
@@ -0,0 +1,334 @@
+
+## Add Blazor DialogProvider Component
+
+* Open **~/_Imports.razor** file or any other page under the `~/Pages` folder where the component is to be added and import the **Syncfusion.Blazor.Popups** namespace.
+
+{% tabs %}
+{% highlight razor tabtitle="~/Imports.razor" %}
+
+@using Syncfusion.Blazor
+@using Syncfusion.Blazor.Popups
+
+{% endhighlight %}
+{% endtabs %}
+
+* Now, add the Syncfusion DialogProvider Component in **~/_MainLayout.razor** file.
+
+{% tabs %}
+{% highlight razor tabtitle=".NET 6 (~/_MainLayout.razor)" %}
+
+<Syncfusion.Blazor.Popups.SfDialogProvider/>
+
+{% endhighlight %}
+{% endtabs %}
+
+## Available predefined dialogs
+
+The dialog component is used to render the `Alert`, `Confirm`, and `Prompt` dialogs with minimal code. The alert, confirm, and prompt dialogs are shown using [DialogServices](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html).
+
+There are three types of predefined dialogs available:
+
+* [Alert](#show-alert-dialog)
+* [Confirm](#show-confirm-dialog)
+* [Prompt](#show-prompt-dialog)
+
+### Show alert dialog
+
+An `Alert` dialog box is used to display errors, warnings, and information that needs user awareness, this can be achieved by using [DialogService.AlertAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_AlertAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) method. The `Alert` dialog display along with ‘Ok’ button.But, it does not has any specific return type. When you click on ‘Ok’ button then the `Alert` method will complete.
+
+Use the following code to render a simple alert dialog in an application:
+
+{% tabs %}
+{% highlight razor %}
+
+{% include_relative code-snippet/alert-dialog.razor %}
+
+{% endhighlight %}
+{% endtabs %}
+
+![Alert Dialog](./images/blazor-alert-dialog.png)
+
+### Show confirm dialog
+
+A `Confirm` dialog displays a specified message along with ‘OK’ and ‘Cancel’ button where it returns a boolean value according to the user's click action, this can be achieved by using [DialogService.ConfirmAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_ConfirmAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) method. It is used to get approval from a user and it appears before any critical action.
```

Comment:
A `Confirm` dialog displays a specified message along with the ‘OK’ and ‘Cancel’ buttons, which returns a boolean value according to the user's click action. This can be achieved by using the [DialogService.ConfirmAsync](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Popups.SfDialogService.html#Syncfusion_Blazor_Popups_SfDialogService_ConfirmAsync_System_String_System_String_Syncfusion_Blazor_Popups_DialogOptions_) method. It is used to get approval from the user, and it appears before any critical action.

Reply:
Addressed the review comments